website/docs/components/code-block/partials/guidelines/guidelines.md

```diff
@@ -0,0 +1,96 @@
+## Usage
+
+### When to use
+
+- When displaying code examples and longer snippets of code that benefit from syntax highlighting.
+
+### When not to use
+
+- As a full-blown code editor.
+
+### When to use a CodeBlock vs a CopySnippet
```

Comment:
On the [other documentation page](https://helios.hashicorp.design/components/copy/snippet), is called "Copy Snippet"
```suggestion
### When to use a CodeBlock vs a Copy Snippet
```